doc/classes/OS.xml

```diff
@@ -216,6 +216,15 @@
 				Not to be confused with [method get_user_data_dir], which returns the [i]project-specific[/i] user data path.
 			</description>
 		</method>
+		<method name="get_distribution_name" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the name of the distribution for Linux and BSD platforms (e.g. Ubuntu, Manjaro, OpenBSD, etc.).
+				Returns the same value as [method get_name] for standard Android. But attempts to return the custom ROM name for Android derivatives.
+				Returns the same value as [method get_name] for other platforms.
+				[b]Note:[/b] This method is not supported on the web platform.
```

Comment:
What happens on the web platform? Is the platform name returned ("Web"), or is an empty string returned? This should be documented :slightly_smiling_face:

Reply:
Indeed, an empty string. Was added